nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -84,6 +84,16 @@ in {
       default = 2;
       description = "Log level value between 0 (DEBUG) and 4 (FATAL).";
     };
+    logType = mkOption {
+      type = types.enum [ "syslog" "file" "systemd" "errorlog" ];
+      default = "syslog";
+      description = "Log driver used for nextcloud logging. Possible options: errorlog, file, syslog or systemd (see https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/logging_configuration.html).";
+    };
+    logFile = mkOption {
+      type = types.str;
```

Comment:
I'd prefer to make this `types.nullOr types.str` and set it to `null` by default.

Reply:
agree, change it as you like, you know the best practices ;-)

Reply:
If we set this to `null` by default, we also have to amend the strings below `${optionalString ...}`.

Reply:
I'm open to everything and would be glad if this could be solved soon, because everything is an improvement 😉